ui/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
 	"extends": "../tsconfig.base.json",
 	"compilerOptions": {
-		"baseUrl": ".",
+		"rootDir": ".",
```

Comment:
Corrects TypeScript's auto-imports to use relative paths instead of absolute from the UI folder.